packages/web/src/pages/sign-up-page/pages/FinishProfilePage.tsx

```diff
@@ -163,8 +164,7 @@ export const FinishProfilePage = () => {
               name='displayName'
               label={messages.displayName}
               placeholder={messages.inputPlaceholder}
-              required
```

Comment:
should this no longer be required?

Reply:
so in the design it doesn't have the `*` in the label. its still required on a formik level though